MinPlatformPkg/Test/Library/TestPointCheckLib/TraditionalMmCheckLoadedImage.c

```diff
@@ -0,0 +1,27 @@
+/** @file
+
+Copyright (c) 2017, Intel Corporation. All rights reserved.<BR>
+Copyright (c) 2022, Microsoft Corporation. All rights reserved.<BR>
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <PiSmm.h>
+#include <Library/TestPointCheckLib.h>
```

Comment:
Probably do not need the rest of the inclusions anymore?

Reply:
Correct.  I removed them.